Build failure on intel_s1000_crb board for test case:” tests/kernel/smp”

**Describe the bug**
The case:” tests/kernel/smp” was been built failure, but the case: ”samples/hello_world/”can work well, including west build and west flash. 

**To Reproduce**
Steps to reproduce the behavior:
1. west build -b intel_s1000_crb -p auto tests/kernel/smp
2. see error

**Expected behavior**
Build successfully on intel_s1000_crb board for test case:” tests/kernel/smp”

**Screenshots or console output**
```
[chenpeng@localhost zephyr]$ west build -b intel_s1000_crb -p auto tests/kernel/smp
-- west build: making build dir /home/chenpeng/zephyrproject/zephyr/build pristine
-- west build: generating a build system
Including boilerplate (Zephyr base): /home/chenpeng/zephyrproject/zephyr/cmake/app/boilerplate.cmake
-- Application: /home/chenpeng/zephyrproject/zephyr/tests/kernel/smp
-- Zephyr version: 2.2.99 (/home/chenpeng/zephyrproject/zephyr)
-- Found Python3: /usr/bin/python3.6 (found suitable exact version "3.6.8") found components:  Interpreter
-- Board: intel_s1000_crb
-- Found west: /home/chenpeng/.local/bin/west (found suitable version "0.7.2", minimum required is "0.7.1")
-- Found toolchain: xcc (/home/chenpeng/xtensa)
-- Found BOARD.dts: /home/chenpeng/zephyrproject/zephyr/boards/xtensa/intel_s1000_crb/intel_s1000_crb.dts
-- Generated zephyr.dts: /home/chenpeng/zephyrproject/zephyr/build/zephyr/zephyr.dts
-- Generated devicetree_unfixed.h: /home/chenpeng/zephyrproject/zephyr/build/zephyr/include/generated/devicetree_unfixed.h
Parsing /home/chenpeng/zephyrproject/zephyr/Kconfig
Loaded configuration '/home/chenpeng/zephyrproject/zephyr/boards/xtensa/intel_s1000_crb/intel_s1000_crb_defconfig'
Merged configuration '/home/chenpeng/zephyrproject/zephyr/tests/kernel/smp/prj.conf'
Configuration saved to '/home/chenpeng/zephyrproject/zephyr/build/zephyr/.config'
Kconfig header saved to '/home/chenpeng/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h'
-- The C compiler identification is Clang 3.9.0
-- The CXX compiler identification is Clang 3.9.0
-- The ASM compiler identification is Clang
-- Found assembler: /home/chenpeng/xtensa/XtDevTools/install/tools/RI-2018.0-linux/XtensaTools/bin/xt-clang
-- Cache files will be written to: /home/chenpeng/.cache/zephyr
-- Configuring done
-- Generating done
-- Build files have been written to: /home/chenpeng/zephyrproject/zephyr/build
-- west build: building application
[1/116] Preparing syscall dependency handling

[106/116] Building C object zephyr/soc/xtensa/intel_s1000/CMakeFiles/soc__xtensa__intel_s1000.dir/soc_mp.c.obj
FAILED: zephyr/soc/xtensa/intel_s1000/CMakeFiles/soc__xtensa__intel_s1000.dir/soc_mp.c.obj
ccache /home/chenpeng/xtensa/XtDevTools/install/tools/RI-2018.0-linux/XtensaTools/bin/xt-clang -DBUILD_VERSION=zephyr-v2.2.0-1979-g6627249dc1e1 -DKERNEL -D_FORTIFY_SOURCE=2 -D__ZEPHYR__=1 -I../drivers -I../include -Izephyr/include/generated -I../soc/xtensa/intel_s1000 -I../soc/xtensa/intel_s1000/include -I../subsys/testsuite/include -I../subsys/testsuite/ztest/include -isystem ../lib/libc/minimal/include -Os -imacros/home/chenpeng/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -fms-extensions -imacros/home/chenpeng/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wno-sometimes-uninitialized -Wno-shift-overflow -Wno-missing-braces -Wno-self-assign -Wno-unused-function -Wno-initializer-overrides -Wno-section -Wno-unknown-warning-option -Wno-unused-variable -Wno-format-invalid-specifier -Wno-gnu -Wno-tautological-compare -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-strict-overflow -ffunction-sections -fdata-sections -mlongcalls -std=c99 -MD -MT zephyr/soc/xtensa/intel_s1000/CMakeFiles/soc__xtensa__intel_s1000.dir/soc_mp.c.obj -MF zephyr/soc/xtensa/intel_s1000/CMakeFiles/soc__xtensa__intel_s1000.dir/soc_mp.c.obj.d -o zephyr/soc/xtensa/intel_s1000/CMakeFiles/soc__xtensa__intel_s1000.dir/soc_mp.c.obj   -c /home/chenpeng/zephyrproject/zephyr/soc/xtensa/intel_s1000/soc_mp.c
/home/chenpeng/zephyrproject/zephyr/soc/xtensa/intel_s1000/soc_mp.c:153:13: error: use of undeclared identifier 'LP_SRAM_BASE'
        *((u32_t *)LPSRAM_BOOT_VECTOR_ADDR) = (u32_t)__start;
                   ^
../soc/xtensa/intel_s1000/memory.h:76:36: note: expanded from macro 'LPSRAM_BOOT_VECTOR_ADDR'
#define LPSRAM_BOOT_VECTOR_ADDR                 (LP_SRAM_BASE + 0x08)
                                                 ^
1 error generated.
[108/116] Building C object zephyr/kernel/CMakeFiles/kernel.dir/timer.c.obj
ninja: build stopped: subcommand failed.
FATAL ERROR: command exited with status 1: /usr/bin/cmake --build /home/chenpeng/zephyrproject/zephyr/build
```

**Environment (please complete the following information):**
 •	OS :  fedora28
•	XTENSA_TOOLCHAIN_PATH :  /home/chenpeng/xtensa
•	ZEPHYR_TOOLCHAIN_VARIANT :  xcc
•	TOOLCHAIN_VER : RI-2018.0-linux
•	XTENSA_CORE :  X6H3SUE_RI_2018_0
•	XTENSA_SYSTEM :  ${XTENSA_TOOLCHAIN_PATH}/XtDevTools/install/tools/RI-2018.0-linux/XtensaTools/config/
•	XTENSA_BUILD_PATHS :  ${XTENSA_TOOLCHAIN_PATH}/XtDevTools/install/builds/
•	XTENSA_OCD_PATH :  /opt/tensilica/xocd-12.0.4
•	ZEPHYR_SDK_INSTALL_DIR : /home/chenpeng/zephyr-sdk-0.11.1/
•	Commit ID  : 6627249dc1e1dc3d667081717e2f8935279939d7

--- a/soc/xtensa/intel_s1000/memory.h
+++ b/soc/xtensa/intel_s1000/memory.h
@@ -73,6 +73,6 @@
 #define LPRAM_SIZE				(DT_REG_SIZE(DT_NODELABEL(sram1)))
 
 /* Boot vector resideing in LP-SRAM for core #1 */
-#define LPSRAM_BOOT_VECTOR_ADDR			(LP_SRAM_BASE + 0x08)
+#define LPSRAM_BOOT_VECTOR_ADDR			(LPRAM_BASE + 0x08)
 
 #endif /* __INC_MEMORY_H */

